youths/models.py

```diff
@@ -43,12 +44,12 @@ class YouthProfile(models.Model):
     )  # For documenting e.g. restrictions on gaming or premises
 
     # Permissions
-    approved_by = models.ForeignKey(
-        Profile,
-        related_name="approved_youth_profiles",
-        null=True,
-        blank=True,
-        on_delete=models.SET_NULL,
+    approver_email = models.EmailField()
+    approval_token = models.CharField(
+        max_length=36, blank=True, default=uuid.uuid4, editable=False
```

Comment:
I was considering it also, but ATM since there isn't an expiration time for tokens, they're invalidated by removing the token (setting the value to be `""`). `blank=True` and `unique=True` won't work together, right?

Reply:
If you guys recommend it, I could
- remove the `blank=True` and add `unique=True`
- add an expiration time
- always check an incoming token for expiration

I think this would mean that a token couldn't be removed so the stale token would be stored in the YouthProfile object until it's refreshed.

I recommend that we leave it as is and later pluck the token into it's own model.

Reply:
> I was considering it also, but ATM since there isn't an expiration time for tokens, they're invalidated by removing the token (setting the value to be `""`). `blank=True` and `unique=True` won't work together, right?

Actually with the uniqueness constraint it needs to have also `null=True`. That would probably be my preferred solution, you could invalidate the token by setting it null.

Reply:
Your suggestion has been taken into consideration. The suggestion was rejected. Thank you for your suggestion.

😄 We'll refine the permissions/tokens feature in a future PR.

Reply:
😄